.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- This template is for Devs to give QA details before moving the issue To-Test -->
+
+<!-- Link to the issue -->
+
+### Root cause
```

Comment:
It's not really a `cause` its more like a solution. So this could be defined as summary. and then place holder for the issue number so it can be referenced e.g. 
```
### Summary
Issue: #
<!-- Define, summarise what feature is added or fixed -->
```

Reply:
Given Neil's github action is now pretty much proven we can switch back to using standard issue linking, so suggest something like `Fixes #`